packages/common/container-definitions/src/test/types/validateContainerDefinitionsPrevious.generated.ts

```diff
@@ -900,30 +900,6 @@ declare function use_old_InterfaceDeclaration_IFluidPackageEnvironment(
 use_old_InterfaceDeclaration_IFluidPackageEnvironment(
     get_current_InterfaceDeclaration_IFluidPackageEnvironment());
 
-/*
```

Comment:
revert?

Reply:
No, these changes were needed per the ci build. My hunch is that the types changes changed what ts-morph enumerates from the previous package. But the type tests are in a weird state right now so I'm not sweating the details yet.